drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -483,7 +483,7 @@ int wifi_nrf_wpa_supp_scan2(void *if_priv, struct wpa_driver_scan_params *params
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
 
 	if (vif_ctx_zep->scan_in_progress) {
-		LOG_INF("%s: Scan already in progress\n", __func__);
+		LOG_DBG("%s: Scan already in progress\n", __func__);
```

Comment:
These (this and below except `successfully` logs) are errors, so, would be good to leave them as is.

Reply:
If they are errors, shouldn't they rather be set to `ERR`?

Reply:
Yes, they should be `LOG_ERR`